### ISSUE TITLE
Cannot make request to mapbox anymore

### ISSUE THREAD

**Issue by vinhtq** [perliedman/lrm-mapbox#1]
Could you please check because i get the error from chrome browser:

No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:1337' is therefore not allowed access

Even example you wrote cannot run

**Comment by perliedman** [comment]
It appears Mapbox does a redirect (HTTP 301) for `http` requests to `https` now, and the redirect doesn't include the `Access-Control-Allow-Origin` header, resulting in this error.

Thanks for notifying me, I guess it should be simple to just always use `https` instead.

Temporary workaround is to explicitly use HTTPS, using this option:

``` javascript
L.Routing.control({
    router: new L.Routing.Mapbox('your mapbox access token', {
        serviceUrl: 'https://api.tiles.mapbox.com/v4/directions/'
    }),
}).addTo(map);
```

**Comment by vinhtq** [comment]
Thank you! It helps me alot

**Comment by vinhtq** [comment]
Thank you!

**Comment by perliedman** [comment]
Fixed, and published a new version 1.0.2.
